### PR TITLE
added EIB example config file to create an iso with longhorn included

### PR DESCRIPTION
**Pull request by dprodanov4**

### CODE REVIEW

File: asciidoc/components/longhorn.adoc [hunk 1]
```diff
@@ -201,4 +202,53 @@ kubectl -n longhorn-system get svc
...
+  packages:
+    sccRegistrationCode: <reg-code>
+    packageList:
+      - open-iscsi
+  users:
+  - username: root
+    encryptedPassword: <password>
+----
+
+* After the image is build you can use it to install your OS and Longhorn will be deployed on top of RKE2.
```
Comment (e-minguez) [inline asciidoc/components/longhorn.adoc:254]: Would you mind adding how to add extra values to the chart? Usually no one deploys charts with default values :D

File: asciidoc/components/longhorn.adoc [hunk 2]
```diff
@@ -201,4 +202,53 @@ kubectl -n longhorn-system get svc
...
 
+== Use Edge Image Builder to deploy Longhorn
 
+The config file from below can be used to create an image via https://github.com/suse-edge/edge-image-builder[EIB] with Longhorn being setup as well.
+
+* Build the container
+
+[,console]
+----
+podman build -t eib:dev .
```
Comment (atanasdinov) [inline asciidoc/components/longhorn.adoc:213]: This must be dropped in favour of using our IBS build, however, the Helm integration is not yet included in the latest release candidate (RC1).

CC: @agracey

**Review by atanasdinov — APPROVED** [review]

File: asciidoc/components/longhorn.adoc (hunk 1 above)
Comment (rdoxenham) [reply asciidoc/components/longhorn.adoc:254]: Also, do we have special handling for Longhorn air-gapped images now that we should mention?

Comment (atanasdinov) [reply asciidoc/components/longhorn.adoc:254]: This is currently a non-airgapped guide.

Comment (dprodanov4) [reply asciidoc/components/longhorn.adoc:254]: We still don’t have an air-gapped solution for their images.

Comment (agracey) [reply asciidoc/components/longhorn.adoc:271]: All of the rancher projects publish a list of images that need to be pulled for airgapping. Would that help us here?

Comment (agracey) [reply asciidoc/components/longhorn.adoc:271]: That said, I could be comfortable calling that out of scope for this release with a plan to add a note to the docs about how to run LH airgapped in a 3.0.z release

File: asciidoc/components/longhorn.adoc [hunk 3]
```diff
@@ -201,4 +202,70 @@ kubectl -n longhorn-system get svc
 
 * Once you have retrieved the longhorn-frontend IP you can start using the UI by navigating to it in your browser.
 
+== Use Edge Image Builder to deploy Longhorn
 
+The config file from below can be used to create an image via https://github.com/suse-edge/edge-image-builder[EIB] with Longhorn being setup as well.
```
[the remark this answers is not in the document]
Comment (dprodanov4) [reply asciidoc/components/longhorn.adoc:207]: I agree with you and was going to update it once we have the `./edge-image-builder.adoc` updated

**Review by atanasdinov — APPROVED** [review]

File: asciidoc/components/longhorn.adoc (hunk 1 above)
Comment (agracey) [reply asciidoc/components/longhorn.adoc:271]: 👍

**Review by agracey — APPROVED** [review]

**Pull request merged**